client/electron/src/index.ts

```diff
@@ -46,6 +46,7 @@ if (!lock) {
 
   // Run Application
   (async (): Promise<any> => {
+    let sigintProcessed = false;
```

Comment:
The callback is being called twice for some reason, I added this boolean to avoid cleaning up Parsec twice.